### PR TITLE
feat(dedicated.network-security): remove old link for statistics

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-496
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

- remove old link for statistics
- add new link to network security dashboard
- re-order menu links for network security dashboard
- add link for network security dashboard into hosted private cloud

### CODE REVIEW

**Review by anooparveti — DISMISSED** [review]
@Steffy29 I see that you have removed a controller and template file. Can you also please clean-up the content used in this page in trans files (only in `Messages_fr_FR.json` of course)?

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.controller.js [hunk 1]
```diff
@@ -452,6 +452,10 @@ export default class IpListController {
         );
     };
 
+    $scope.displayStatistics = function displayStatistics(ipBlock) {
+      self.goToStatistics(ipBlock);
+    };
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/list.controller.js:457]: No need to manage tracking anymore ? As before you managed it directly in the template 
`data-ng-click="setAction('ip/mitigation/statistics/ip-ip-mitigation-statistics', { ipBlock: ipBlock, ip: ip ,tracking : tracking['statistics']})"`

If not maybe you can simplify as 

```suggestion
    $scope.displayStatistics = self.goToStatistics;
```

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js [hunk 2]
```diff
@@ -2,7 +2,8 @@ import { PERIODS, PERIOD_LIST } from './scrubbing-center.constant';
 
 export default class ScrubbingCenterController {
   /* @ngInject */
-  constructor($translate, Alerter, networkSecurityService) {
+  constructor($scope, $translate, Alerter, networkSecurityService) {
+    this.$scope = $scope;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:6]: **nitpick:** Avoid using `$scope` and use Controller object for storing data

Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:6]: **nitpick:** Avoid using `$scope` and use Controller object for storing data

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.controller.js (hunk 1 above)
Comment (Steffy29) [reply packages/manager/apps/dedicated/client/app/ip/components/list/list.controller.js:457]: For the moment the tracking is not set for the new page Network Security Dashboard.

**Review by darsene — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**